allennlp/modules/token_embedders/bidirectional_language_model_token_embedder.py

```diff
@@ -1,26 +1,21 @@
-import json
-from typing import Dict, Tuple, TYPE_CHECKING
+from typing import Tuple
+import warnings
 
-import torch
-
-from allennlp.common.checks import ConfigurationError
-from allennlp.data import TokenIndexer, Token
-from allennlp.modules.scalar_mix import ScalarMix
 from allennlp.modules.token_embedders.token_embedder import TokenEmbedder
-from allennlp.nn.util import remove_sentence_boundaries, get_text_field_mask, add_sentence_boundary_token_ids
-
-# Importing at runtime results in a circular dependency.
-if TYPE_CHECKING:
-    from allennlp.models.bidirectional_lm import BidirectionalLanguageModel
 
 
 @TokenEmbedder.register('bidirectional_lm_token_embedder')
 class BidirectionalLanguageModelTokenEmbedder(TokenEmbedder):
+    # pylint: disable=abstract-method
```

Comment:
This shouldn't be needed. Looks like the problem is that we aren't extending the `LanguageModelTokenEmbedder` in the line above. This is good example of why keeping the tests is around is prudent. Fortunately you should be able to do this with minimal duplication. See https://github.com/allenai/allennlp/blob/511c846c9c779fdac0066dbae6765c1619037098/allennlp/tests/fixtures/bidirectional_lm/experiment.jsonnet for how to share config across jsonnet files if you aren't already familiar.